Sources/ViewControllers/Settings/SettingsTableViewController.swift

```diff
@@ -37,7 +37,7 @@ class SettingsTableViewController: UITableViewController, MFMailComposeViewContr
         case .item:
             return "settings.tab-bar.item".localized
         case .information:
-            return "settings.sections.information".localized
+            return "settings.sections.discover".localized
```

Comment:
Changing this has changed the header for the English language, and this has to be changed for all the languages.